src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToGrid/MoveToGrid.java

```diff
@@ -0,0 +1,16 @@
+package edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid;
+
+import edu.greenblitz.tobyDetermined.commands.swerve.MoveToPos;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class MoveToGrid extends MoveToPos {
```

Comment:
Raz dont like Pos I think

Reply:
Not only Raz...

Reply:
it'll change when i merge master into this